src/applications/hca/actions.js

```diff
@@ -9,7 +10,13 @@ export function submitIDForm(formData) {
   return dispatch => {
     dispatch({ type: SUBMIT_ID_FORM_STARTED });
 
-    const baseUrl = '/health_care_applications/enrollment_status';
+    // We need bypass the local API when running locally, so hit dev-api
```

Comment:
I'm not sure if this is something we have done in the past, but this seems to be better than manually changing the value of `baseUrl` when running locally and wanting to get a non-500 response from the `enrollement_status` endpoint.